### PR TITLE
contenteditable support

### PR DESCRIPTION
**Pull request by tpict**
This PR merges @davidsulc's work in #113 with the master branch. I have elected to add a new InputReader/Writer for handling `contenteditable` elements instead of extending the default.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.7.0",
```
Comment (scott-w) [inline bower.json]: Bump this to 0.8.0

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.7.0",
+  "version": "0.7.1",
```
Comment (scott-w) [inline package.json]: And here

File: src/backbone.syphon.js [hunk 3]
```diff
@@ -149,7 +153,7 @@ var getForm = function(viewOrForm) {
   if (_.isUndefined(viewOrForm.$el)) {
     return $(viewOrForm).find(':input');
   } else {
-    return viewOrForm.$(':input');
+    return viewOrForm.$(':input, [contenteditable]');
```
Comment (scott-w) [inline src/backbone.syphon.js]: Have you missed off adding `[contenteditable]` to the `_.isUndefined` case above? ☝️

**Comment by tpict** [comment]
@scott-w done :smile:

**Pull request merged**